packages/db/src/schema/trading/defs.ts

```diff
@@ -235,7 +235,7 @@ export const portfolioSnapshots = tradingComps.table(
     index("idx_portfolio_snapshots_competition_agent_timestamp").on(
       table.competitionId,
       table.agentId,
-      table.timestamp,
+      desc(table.timestamp),
```

Comment:
Now the declared index has the correct sorting modifier, and the other 2 duplicates can be deleted via subabase dashboard ui